Formula/biber.rb

```diff
@@ -540,30 +542,27 @@ def install
       end
     end
 
+    bin_before = Dir[libexec/"bin/*"].to_set
     system "perl", "Build.PL", "--install_base", libexec
     system "./Build"
     system "./Build", "install"
-
-    bin.install Dir[libexec/"bin/*"]
-    bin.env_script_all_files(libexec/"bin", PERL5LIB: ENV["PERL5LIB"])
+    bin_after = Dir[libexec/"bin/*"].to_set
+    (bin_after - bin_before).each do |file|
+      basename = Pathname(file).basename
+      (bin/basename).write_env_script file, PERL5LIB: ENV["PERL5LIB"]
+    end
     man1.install libexec/"man/man1/biber.1"
+    (pkgshare/"test").install "t/tdata/annotations.bcf", "t/tdata/annotations.bib"
```

Comment:
Just picked the first test from directory. Not sure if there is a better one. We've been hit a few times by upstream modifying the `test.bcf` file with new releases. Unless we save a copy, it didn't seem worth seeing CI failures related to checksum mismatches or version incompatibility.

Reply:
I didn't realize there was test directory but that seems like a much more reliable solution.